src/org/parosproxy/paros/security/SslCertificateService.java

```diff
@@ -63,7 +65,7 @@
 	 * @throws UnrecoverableKeyException
 	 * @throws MissingRootCertificateException when it wasn't initialized.
 	 */
-	KeyStore createCertForHost(String hostname)
+	KeyStore createCertForHost(String hostname, InetAddress listeningAddress)

```

Comment:
I'd suggest using a default method instead (to keep binary compatibility).

Reply:
How about `KeyStore createCert(CertData)`? (With `CertData` containing the desired common name and alternative (IPs/domain) names.) That would be more flexible for the caller and allow for future changes without breaking binary compatibility.